### PR TITLE
Support all column types in SignalSchema.from_column_types

### PR DESCRIPTION
**Pull request by dreadatour**
Having registered dataset:

```python
from datachain.lib.dc import DataChain 

DataChain.from_storage("s3://dql-50k-laion-files/").save("laion")
```

Making basic query on it:

```python
from datachain.lib.dc import C, DataChain 

DataChain(name="laion").filter(C.file__path.glob("*.jpg"))
```

Fails with error:

```
Traceback (most recent call last):
  File "<string>", line 4, in <module>
  File "/Users/vlad/work/iterative/datachain/src/datachain/lib/dc.py", line 227, in __init__
    self.signals_schema |= SignalSchema.from_column_types(self.column_types)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/vlad/work/iterative/datachain/src/datachain/lib/signal_schema.py", line 108, in from_column_types
    raise SignalSchemaError(
datachain.lib.signal_schema.SignalSchemaError: signal schema cannot be obtained for column 'sys__rand': unsupported type 'None'
Traceback (most recent call last):
  File "<string>", line 4, in <module>
  File "/Users/vlad/work/iterative/datachain/src/datachain/lib/dc.py", line 227, in __init__
    self.signals_schema |= SignalSchema.from_column_types(self.column_types)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/vlad/work/iterative/datachain/src/datachain/lib/signal_schema.py", line 108, in from_column_types
    raise SignalSchemaError(
datachain.lib.signal_schema.SignalSchemaError: signal schema cannot be obtained for column 'sys__rand': unsupported type 'None'
```

This is because `sys__rand` column type is `UInt64` (at least in SaaS), and this type is not exists in `DATACHAIN_TO_TYPE`, which leads to an error here: https://github.com/iterative/datachain/blob/main/src/datachain/lib/signal_schema.py#L107-L111

### CODE REVIEW

**Review by mattseddon — APPROVED** [review]
is there a regression test somewhere?

File: src/datachain/lib/convert/flatten.py [hunk 1]
```diff
@@ -10,19 +10,25 @@
     Boolean,
     DateTime,
     Float,
+    Float32,
+    Float64,
```
Comment (mattseddon) [inline src/datachain/lib/convert/flatten.py:14]: Does this close https://github.com/iterative/datachain/issues/12 ?

Comment (skshetry) [reply src/datachain/lib/convert/flatten.py:14]: If it does, please don't forget to rollback this change: https://github.com/iterative/datachain/pull/13.

**Review by ilongin — APPROVED** [review]
LGTM

File: src/datachain/lib/convert/flatten.py (hunk 1 above)
Comment (dreadatour) [reply src/datachain/lib/convert/flatten.py]: This doesn't close #12, but it will be easy to fix, I'll make separate PR for that.

**Comment by dreadatour** [comment]
> is there a regression test somewhere?

Wasn't sure about this approach. Updated the PR with better code, added test, please, take a look 🙏

**Review by dtulga — APPROVED** [review]
LGTM, thanks!

**Review by mattseddon — APPROVED** [review]

**Pull request merged**